contracts/BasicMultiToken.sol

```diff
@@ -12,118 +12,127 @@ contract BasicMultiToken is Ownable, StandardToken, DetailedERC20, ERC1003Token,
     using CheckedERC20 for ERC20;
     using CheckedERC20 for DetailedERC20;
 
-    uint internal inLendingMode;
-    bool public bundlingEnabled = true;
+    ERC20[] internal _tokens;
+    uint internal _inLendingMode;
+    bool internal _bundlingEnabled = true;
```

Comment:
_bundlingEnabled and _tokens can be private.
Also add public getter for _bundlingEnabled.